functional_tests/test_issue_879.py

```diff
@@ -0,0 +1,52 @@
+import os
+import sys
+import unittest
+from subprocess import PIPE, Popen
+
+import nose
+from nose.plugins.skip import SkipTest
+
+
+support = os.path.join(os.path.dirname(__file__), 'support', 'issue879')
+
+class TestIssue879(unittest.TestCase):
```

Comment:
Why not use the `PluginTester` like many of the other tests?  It already does some of the work you're doing.